packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -225,33 +227,31 @@ class MinimumTextContrastGuideline extends AccessibilityGuideline {
 
       // We need to look up the inherited text properties to determine the
       // contrast ratio based on text size/weight.
-      double fontSize;
+      double? fontSize;
       bool isBold;
-      final String text = (data.label?.isEmpty == true) ? data.value : data.label;
+      final String text = data.label.isEmpty ? data.value : data.label;
       final List<Element> elements = find.text(text).hitTestable().evaluate().toList();
       Rect paintBounds;
       if (elements.length == 1) {
         final Element element = elements.single;
-        final RenderBox renderObject = element.renderObject as RenderBox;
-        element.renderObject.paintBounds;
+        final RenderBox renderObject = element.renderObject! as RenderBox;
```

Comment:
yikes, do we document / check that this is only used with RenderBoxes?

Reply:
I added an assert. Currently, it is guaranteed that the renderObject will always be a RenderBox because the element finder above only returns Elements of `Text` or `Editable` widgets, which will always have RenderBox elements.